src/ExadelMentorship.IntegrationTests/Features/WeatherTest.cs

```diff
@@ -0,0 +1,25 @@
+﻿using ExadelMentorship.BusinessLogic.Exceptions;
+using ExadelMentorship.BusinessLogic.Features.WeatherFeature;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ExadelMentorship.IntegrationTests.Features
+{
+    public class WeatherTest
+    {
+        [Fact]
+        public async Task GetTemperatureByCityName_WhenCityNameIsNotCorrect_ThrowsException()
+        {
+            //Arrange
+            Weather weather = new Weather(new HttpClient());
+
+            //Act and Assert
+            await Assert.ThrowsAsync<NotFoundException>(() => weather.GetTemperatureByCityName("AAA"));
+        }
```

Comment:
You need `happy path` test case. It returns a number and does not throw.

Reply:
Done